Sources/AppStoreConnectCLI/Commands/BundleIds/ListBundleIdsCommand.swift

```diff
@@ -17,64 +15,32 @@ struct ListBundleIdsCommand: CommonParsableCommand {
     @Option(help: "Limit the number of resources (maximum 200).")
     var limit: Int?
 
-    @OptionGroup()
-    var filters: FilterOptions
-
-    func run() throws {
-        let service = try makeService()
+    @Option(parsing: .upToNextOption, help: "Filter the results by reverse-DNS bundle ID identifier (eg. com.example.app)")
+    var filterIdentifier: [String]
 
-        let request = APIEndpoint.listBundleIds(
-            filter: [BundleIds.Filter](filters),
-            limit: limit
-        )
+    @Option(parsing: .upToNextOption, help: "Filter the results by app name")
+    var filterName: [String]
 
-        let bundleId = try service.request(request)
-            .map { $0.data.map(BundleId.init) }
-            .await()
-
-        bundleId.render(format: common.outputFormat)
-    }
-}
+    @Option(
+        parsing: .upToNextOption,
+        help: "Filter the results by platform (\(Platform.allCases.description))."
+    )
+    var filterPlatform: [String]
```

Comment:
Do we want to have `[String]` here and move the `enum Platform` validation into the operation?

Reply:
At the moment there's a conversion and incorrect params are ignored. We might want to have a `Platform` type sitting in `Model` that we convert from when we call the API. Even though that's some duplication :/